spec/support/helpers/general_spec_helpers.rb

```diff
@@ -50,4 +61,18 @@ def with_env(vars)
   ensure
     vars.each_pair { |k, _| ENV.delete(k) }
   end
+
+  # decorate objects
+  def decorate(obj)
+    Odk::DecoratorFactory.decorate(obj)
+  end
```

Comment:
I love the initiative you've taken here, to move this into the general helpers file!

There is an issue here though. In fact two issues. 

1. There is an issue with naming specificity/scope. What is it?
2. There is an issue with cohesion. See https://en.wikipedia.org/wiki/Cohesion_(computer_science). In general, files like 'general ____' are a bad idea. We should probably get rid of this one even, because it tends to attract the kitchen sink (i.e. a barely cohesive set of methods that do all kinds of things). This is related to the fat models thing. Fat nothing, skinny everything! "Fatness" generally means lower cohesion, too much responsibility, etc. A better way to solve the problem of "where to put supporting methods for specs" is to use contexts. In this case, perhaps a context called "odk rendering". Are there other methods in the general file we could move in there? (I know there is an odk submission context but I don't want to mess with that one because Devney's branch is heavily modifying it, and also I think the rendering stuff is separate from the submission stuff). Remember that once you define the context, you need to include it in each place it's used. See existing code for how to do that.